src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -64,7 +64,8 @@ public static bool IsZipDeployment(this IEnvironment environment)
         {
             return !string.IsNullOrEmpty(environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteZipDeployment)) ||
                 !string.IsNullOrEmpty(environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteAltZipDeployment)) ||
-                !string.IsNullOrEmpty(environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteRunFromPackage));
+                !string.IsNullOrEmpty(environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteRunFromPackage)) ||
+                !string.IsNullOrEmpty(environment.GetEnvironmentVariable(EnvironmentSettingNames.ScmBuildPackage));
```

Comment:
This should only be evaluated in Linux consumption scenarios, correct?

Reply:
Yes. only supported for linux consumption.